app/screens/image_preview/downloader.android.js

```diff
@@ -37,18 +38,29 @@ export default class Downloader extends PureComponent {
         show: false
     };
 
+    static contextTypes = {
```

Comment:
We're just going all in on the context route now, aren't we. I'm impressed we've managed to add it to around 4 different PRs since this morning 😛